container/lxd/export_test.go

```diff
@@ -40,6 +40,7 @@ func PatchGenerateVirtualMACAddress(patcher patcher) {
 	})
 }
 
+
```

Comment:
Does an extra line-break really make gofmt sad?

Reply:
anything that go fmt would change in a file makes it sad.

Reply:
(the bot doesn't have a way to push changes back to github, it just approves github for doing the merge, so anything that we would want to be done (run go fmt over the code base) needs to be done by users)